Quadriga/src/main/java/edu/asu/spring/quadriga/web/workspace/ActivateWSController.java

```diff
@@ -57,114 +54,49 @@ protected void initBinder(WebDataBinder binder) {
         binder.setValidator(validator);
     }
 
-    /**
-     * This calls workspaceManger to list the workspace associated with a
-     * project to activate.
-     * 
-     * @param model
-     * @param projectid
-     * @return String - URL of the form
-     * @throws QuadrigaStorageException
-     * @author Kiran Kumar Batna
-     */
+	/**
+	 * This method activates workspace for a given workspaceid and projectid.
+	 * 
+	 * @param workspaceid
+	 * @param projectid
+	 * @param principal
+	 * @param redirectAttributes
+	 * @return String - URL of the form
+	 * @throws QuadrigaStorageException
+	 * @throws QuadrigaAccessException
+	 * @author Hari Chandana Kanchanapally
+	 */
     @AccessPolicies({
-            @ElementAccessPolicy(type = CheckedElementType.PROJECT, paramIndex = 1, userRole = {
+            @ElementAccessPolicy(type = CheckedElementType.WORKSPACE, paramIndex = 1, userRole = {
                     RoleNames.ROLE_COLLABORATOR_ADMIN,
                     RoleNames.ROLE_PROJ_COLLABORATOR_ADMIN,
                     RoleNames.ROLE_PROJ_COLLABORATOR_CONTRIBUTOR }),

```

Comment:
I believe those roles are wrong. They should be the corresponding workspace roles.